tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c

```diff
@@ -0,0 +1,442 @@
+
+	fixture->start_param.param[0].addr.type = BT_ADDR_LE_RANDOM + 1;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_invalid_sid)
+{
+	int err;
+
+	init_start_param(fixture);
+
+	err = bt_cap_commander_register_cb(&mock_cap_commander_cb);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	fixture->start_param.param[0].adv_sid = BT_GAP_SID_MAX + 1;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_pa_interval_low)
+{
+	int err;
+
+	init_start_param(fixture);
+
+	err = bt_cap_commander_register_cb(&mock_cap_commander_cb);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	fixture->start_param.param[0].pa_interval = BT_GAP_PER_ADV_MIN_INTERVAL - 1;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_invalid_broadcast_id)
+{
+	int err;
+
+	init_start_param(fixture);
+
+	err = bt_cap_commander_register_cb(&mock_cap_commander_cb);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	fixture->start_param.param[0].broadcast_id = BT_AUDIO_BROADCAST_ID_MAX + 1;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_no_subgroups)
+{
+	int err;
+
+	init_start_param(fixture);
+
+	err = bt_cap_commander_register_cb(&mock_cap_commander_cb);
+	zassert_equal(0, err, "Unexpected return value %d", err);
+
+	fixture->start_param.param[0].num_subgroups = 0;
+
+	err = bt_cap_commander_broadcast_reception_start(&fixture->start_param);
+	zassert_equal(-EINVAL, err, "Unexpected return value %d", err);
+
+	zexpect_call_count("bt_cap_commander_cb.broadcast_reception_start", 0,
+			   mock_cap_commander_broadcast_reception_start_cb_fake.call_count);
+}
+
+ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_invalid_nr_subgroups)
```

Comment:
suggest to use 
```suggestion
ZTEST_F(cap_commander_test_broadcast_reception, test_commander_reception_start_invalid_num_subgroups)
```
To better match the name of the field